jest.setup.js

```diff
@@ -1 +1,3 @@
 import '@testing-library/jest-dom';
+import "@testing-library/jest-dom/extend-expect";
```

Comment:
```suggestion
import '@testing-library/jest-dom/extend-expect';
```
single quote로 수정해주세여

Reply:
테스트 코드는 별로의 PR로 가져가는건 어떤가요? 지금 안 맞아보이네요 @dididy @BKJang

Reply:
jest 설정을 위해 새로 만들어둔 PR에 반영하였습니다.